src/components/article/style.css

```diff
@@ -80,7 +80,8 @@ h1 {
 
 .articleFullPageContainer {
     text-align: center;
-    font-family: SourceSansLight;
+    /* commenting this out as it prevents the bold/italics etc styling on the article text being applied */
+    /*font-family: SourceSansLight;*/
```

Comment:
yes there are separate fonts for bold, italics etc which won't work with the wysiwyg editor output